### DISCUSSION TITLE
Hide progress bar

### DISCUSSION THREAD

**Discussion by gponty** [deployphp/deployer#3853] [Help needed]
Hi,

Is it possible to hide progress bar ?
Because in github action it's ugly : 

![image](https://github.com/deployphp/deployer/assets/34276317/2e7f5544-e960-4dcf-9a86-1169338973f0)

and how to not execute login message, by example :

![image](https://github.com/deployphp/deployer/assets/34276317/a3dd575c-bc60-4970-b937-850f2969f9d9)

ty

**Comment by akosglue** [comment]
Can't you just set the CI env variable?
see this line: https://github.com/deployphp/deployer/blob/682ff2171d115bf699839852079832ad26534b8e/src/Executor/Master.php#L206
or is it not feasible for you?

**Comment by gponty** [comment]
~~It doesn't seem to be working, but it might be due to my CI setup. I'll try some other things. Thanks for your response.~~

It Works !!! TY !!
